lib/jsdom/living/post-message.js

```diff
@@ -21,11 +21,11 @@ module.exports = function (message, targetOrigin) {
   }
 
   // TODO: event.source - requires reference to source window
-  // TODO: event.origin - requires reference to source window
   // TODO: event.ports
   // TODO: event.data - structured clone message - requires cloning DOM nodes
   const event = new this.MessageEvent("message", {
-    data: message
+    data: message,
+    origin: targetOrigin
```

Comment:
an origin is needed to have the same behavior and in all cases it should be the same as the targetOrigin (except *), should we do localhost in that case?